quesma/queryparser/pancake_json_rendering.go

```diff
@@ -179,7 +179,11 @@ func (p *pancakeJSONRenderer) combinatorBucketToJSON(remainingLayers []*pancakeM
 		if err != nil {
 			return nil, err
 		}
-		return util.MergeMaps(p.ctx, aggJson, subAggr), nil
+		mergeResult, mergeErr := util.MergeMaps(aggJson, subAggr)
+		if mergeErr != nil {
```

Comment:
I'm wondering if we could propagate this error further without logging it here

Reply:
That'd affect the sub aggregations, I didn't want to touch that at this moment :)